Oxide.Rust/Libraries/Covalence/RustPlayerManager.cs

```diff
@@ -35,6 +38,11 @@ internal void Initialize()
             foreach (var pair in playerData) allPlayers.Add(pair.Key, new RustPlayer(pair.Value.Id, pair.Value.Name));
         }
 
+        /// <summary>
+        /// When a player joins
```

Comment:
I'd probably go with something like... "Register when a player joins" instead, as "When a player joins" doesn't really summarize the method.